src/markitdown/_markitdown.py

```diff
@@ -904,6 +1033,28 @@ def convert(
         elif isinstance(source, requests.Response):
             return self.convert_response(source, **kwargs)
 
+    def convert_url(
+        self, url: str, **kwargs: Any
+    ) -> DocumentConverterResult:  # TODO: fix kwargs type
+        # Handle GitHub issue and pull request URLs directly
+        parsed_url = urlparse(url)
+        if parsed_url.hostname == "github.com" and any(
+            x in parsed_url.path for x in ["/issues/", "/pull/"]
+        ):
+            github_token = kwargs.get("github_token", os.getenv("GITHUB_TOKEN"))
+            if not github_token:
+                raise ValueError(
+                    "GitHub token is required for GitHub issue or pull request conversion."
+                )
+            return GitHubIssueConverter().convert(
+                github_url=url, github_token=github_token
+            )
```

Comment:
@afourney this was the conditional that I added since accessing GH requires API call.